overlord/ifacestate/handlers.go

```diff
@@ -1118,3 +1119,172 @@ func (m *InterfaceManager) doGadgetConnect(task *state.Task, _ *tomb.Tomb) error
 	task.SetStatus(state.DoneStatus)
 	return nil
 }
+
+// doHotplugConnect creates task(s) to (re)create connections in response to hotplug "add" event.
+func (m *InterfaceManager) doHotplugConnect(task *state.Task, _ *tomb.Tomb) error {
+	st := task.State()
+	st.Lock()
+	defer st.Unlock()
+
+	conns, err := getConns(st)
+	if err != nil {
+		return fmt.Errorf("failed to read connections data: %s", err)
```

Comment:
I think we should consistently use `cannot ...` pattern for error messages. How about `cannot retrieve connections`. Alternatively we could just return the error as-is. Was there a specific reason to prefix the returned error?

Reply:
Right the prefix is not really needed here, I missed the fact that getConns already returns a meaningful error message. Removed.